taskcluster/kinds/docker-image/kind.yml

```diff
@@ -22,21 +22,57 @@ tasks:
         args:
             PYTHON_VERSION: "3.8"
             PYTHON_REQ_SUFFIX: ".py38"
-    pushapkscript-python38:
-        definition: pushapkscript
-        parent: python38
-    pushapkscript-python39:
-        definition: pushapkscript
-        parent: python39
-    pushflatpakscript-python38:
-        definition: pushflatpakscript
-        parent: python38
-    pushflatpakscript-python39:
-        definition: pushflatpakscript
-        parent: python39
-    signingscript-python38:
-        definition: signingscript
-        parent: python38
-    signingscript-python39:
-        definition: signingscript
-        parent: python39
+
+    # Used by push-image kind
+    skopeo: {}
+
+    base-image:
+        args:
+            BASE_IMAGE: python:3.9.7
+    # TODO: Figure out PYTHON_REQ_SUFFIX when we decide to have py3.10
+    # base-image-3.10:
+    #     definition: base-image
+    #     args:
+    #         BASE_IMAGE: python:3.10
+    addonscript:
+        parent: base-image
+    balrogscript:
+        parent: base-image
+    beetmoverscript:
+        parent: base-image
+    bouncerscript:
+        parent: base-image
+    githubscript:
+        parent: base-image
+    pushapkscript:
+        parent: base-image
+    pushflatpakscript:
+        parent: base-image
+    pushmsixscript:
+        parent: base-image
+    shipitscript:
+        parent: base-image
+    signingscript:
+        parent: base-image
+    treescript:
+        parent: base-image
+
+    # Testing images
+    pushapkscript-test-py39:
+        definition: base-image-test
+        args:
+            DOCKER_IMAGE_PARENT: python:3.9.7
```

Comment:
Might want to call this BASE_IMAGE to match the above one, and avoid confusion with DOCKER_IMAGE_PARENT that comes from having `parent` in the task definition.